js/phemex.js

```diff
@@ -24,9 +24,10 @@ module.exports = class phemex extends Exchange {
                 'CORS': undefined,
                 'spot': true,
                 'margin': false,
-                'swap': undefined, // has but not fully implemented
+                'swap': true,
                 'future': false,
                 'option': false,
+                'addMargin': undefined,
```

Comment:
I think that this endpoint might be it https://github.com/phemex/phemex-api-docs/blob/master/Public-Contract-API-en.md#change-position-risklimit

Reply:
I'll try this out